### DISCUSSION TITLE
Multiple User Discord Notification

### DISCUSSION THREAD

**Discussion by jrmann** [jef/streetmerchant#1417] [General]
If I wanted to have multiple users tagged in a Discord thread do I just comma separate their ID's like this in the dotenv file? 

Example: DISCORD_NOTIFY_GROUP_3080="<@XXXXXXX>,<@XXXXXXX>"

For other notifications, does anyone know how I could tag @here?

Thanks!

**Comment by ShokoPeak** [comment]
Yeah, you can comma separate the ID's to ping individual users / roles etc. and you can also use @everyone or @here as you would on any Discord post. So you could specifiy something like:

`DISCORD_NOTIFY_GROUP_3080="<@everyone> The 3080 is back in stock!"`

This would post the update with the commented text above and give you something like this in Discord:

![image](https://user-images.githubusercontent.com/76159958/102478850-00d11680-4056-11eb-8751-5f94998209d9.png)

**Comment by jrmann** [comment]
Thanks this is perfect!
